govtool/frontend/src/components/organisms/GovernanceActionDetailsCardData.tsx

```diff
@@ -200,6 +197,19 @@ export const GovernanceActionDetailsCardData = ({
           ),
           visible: type === GovernanceActionType.NewCommittee && !!details,
         },
+        {
+          label: "Parameters",
```

Comment:
```suggestion
          label: "Details",
```
Can we name it details for the user, to avoid confusion with parameter update actions

Reply:
Done :white_check_mark: